src/components/header/ExtendedNav/ExtendedNav.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import classnames from 'classnames'
+
+import { NavCloseButton } from '../NavCloseButton/NavCloseButton'
+import { NavList } from '../NavList/NavList'
+
+// assets
+import closeImg from 'uswds/src/img/close.svg'
+
+type ExtendedNavProps = {
+  primaryItems: React.ReactNode[]
+  secondaryItems: React.ReactNode[]
+  onToggleMobileNav:
```

Comment:
you can add a question mark after the prop name instead of adding `| undefined` and that will have the same effect